projects/storefrontlib/cms-components/misc/icon/icon.model.ts

```diff
@@ -47,6 +47,8 @@ export enum ICON_TYPE {
   EXPAND_ARROWS = 'EXPAND_ARROWS',
   EYE = 'EYE',
   EYE_SLASH = 'EYE_SLASH',
+  FILE = 'FILE',
```

Comment:
This is breaking change. It can be augmented in "account summary".

Reply:
moved to bottom to avoid breaking change
https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes

Reply:
Not a breaking change in the end